part-spec/common/componentID.json

```diff
@@ -106,11 +106,16 @@
                     "example": "2022-03-31",
                     "type": "string"
                 },
+                "eDataheetSpecRevision": {
+                    "description": "revision of the digital datasheet specifications used to create the digital datasheet",
```

Comment:
There's a spelling error here "eDataheetSpecRevision". It should be "eDatasheetSpecRevision"

Reply:
Good Catch. Changed